integration-tests/test/test_dag_correctness.py

```diff
@@ -1,19 +1,113 @@
+                        wait_for_peers_count_at_least(context, validator2, 3)
+                        wait_for_peers_count_at_least(context, validator3, 3)
+
+                        deploy1 = DeployThread("validator1", validator1, contract_path, 10)
+                        deploy1.start()
+
+                        deploy2 = DeployThread("validator2", validator2, contract_path, 10)
+                        deploy2.start()
+
+                        deploy3 = DeployThread("validator3", validator3, contract_path, 10)
+                        deploy3.start()
+
+
+                        deploy1.join()
+                        deploy2.join()
+                        deploy3.join()
+
+                        expected_blocks_count = 31
+                        wait_for_blocks_count_at_least(
+                            context,
+                            validator1,
+                            expected_blocks_count,
+                        )
+                        wait_for_blocks_count_at_least(
+                            context,
+                            validator2,
+                            expected_blocks_count,
+                        )
+                        wait_for_blocks_count_at_least(
+                            context,
+                            validator3,
+                            expected_blocks_count,
+                        )
+
+                        with bootstrap_connected_peer(context=context, bootstrap=bootstrap_node, name='bonded-validator-4', keypair=BONDED_VALIDATOR_KEY_4) as validator4:
+
+                            deploy4 = DeployThread("catch_up", validator1, contract_path, 1)
+                            deploy4.start()
+                            deploy4.join()
+
+                            wait_for_blocks_count_at_least(
+                                context,
+                                validator1,
+                                expected_blocks_count + 1,
+                            )
+                            wait_for_blocks_count_at_least(
+                                context,
+                                validator2,
+                                expected_blocks_count + 1,
+                            )
+                            wait_for_blocks_count_at_least(
+                                context,
+                                validator3,
+                                expected_blocks_count + 1,
+                            )
+                            wait_for_blocks_count_at_least(
+                                context,
+                                validator4,
+                                expected_blocks_count + 1,
+                            )
+
+                            validator1_vdag = validator1.get_vdag()
+                            validator2_vdag = validator2.get_vdag()
+                            validator3_vdag = validator3.get_vdag()
+                            validator4_vdag = validator4.get_vdag()
```

Comment:
I think this is supposed to use `mvdag` and not `vdag` command as per https://github.com/rchain/rchain/pull/2355 and this PR was merged prematurely.

Reply:
Sorry. I would make a quick patch.

Reply:
👍